libs/assets/powershell/iis_global_rewrite.ps1

```diff
@@ -6,7 +6,13 @@
     [parameter(Position=2, Mandatory=$true, HelpMessage="Rewrite type, 'url' or 'farm'")]
     [string]$RewriteType,
     [parameter(Position=3, Mandatory=$true, HelpMessage="URL or farm to redirect matching requests.")]
-    [string]$RedirectUrl
+    [string]$RedirectUrl,
+    [parameter(Position=4, Mandatory=$false, HelpMessage="Conditional Pattern.")]
+    [string]$ConditionalPattern,
+    [parameter(Position=5, Mandatory=$false, HelpMessage="Setting farm conditional IDP first.")]
+    [switch]$ConditionalFirst,
+    [parameter(Position=6, Mandatory=$false, HelpMessage="Setting farm conditional IDP Second.")]
+    [switch]$ConditionalSecond
```

Comment:
Should not be for IDP specifically. For any application that may use the conditional groups.